src/core/OuterExamTimer.jsx

```diff
@@ -1,22 +1,24 @@
 import React, { useEffect, useContext } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
 import PropTypes from 'prop-types';
 import { AppContext } from '@edx/frontend-platform/react';
 import ExamStateContext from '../context';
+import { getLatestAttemptData } from '../data';
 import { ExamTimerBlock } from '../timer';
 import ExamAPIError from '../exam/ExamAPIError';
 import ExamStateProvider from './ExamStateProvider';
 
 const ExamTimer = ({ courseId }) => {
-  const state = useContext(ExamStateContext);
+  const examState = useContext(ExamStateContext);
   const { authenticatedUser } = useContext(AppContext);
-  const {
```

Comment:
[question] Do we just no longer need all of these consts from state?

Reply:
If you take a look at where these were being used in this component, they were just being passed as props to `ExamTimerBlock` and weren't being used anywhere else (except `showTimer`).

I've refactored `ExamTimerBlock` to pull these values either out of Redux (if they're state) or import them out of the `data` module (if they're thunks), which means `ExamTimerBlock` no longer needs to get these values as props. `OuterExamTimer` was getting these as props from _its_ parent and doing some prop drilling to send them to `ExamTimerBlock`, but since `ExamTimerBlock` can now get what it needs itself, it no longer needs `OuterExamTimer` to send it those props.